### DISCUSSION TITLE
Do any of the models allow for L2 Regularization?

### DISCUSSION THREAD

**Discussion by mg10011** [manujosephv/pytorch_tabular#443] [Q&A] [answered]
Basic question to start a conversation.

**Comment by manujosephv** [comment, marked as answer]
Typically, in PyTorch L2 Regularisation is applied using weight decay
Use weight_decay > 0 for L2 regularization:

`optimizer = torch.optim.Adam(model.parameters(), lr=1e-4, weight_decay=1e-5)`

Although this doesn't work like a Lasso Regression and necessarily gives you a sparse usage of features...
